recipes/cairo/all/patches/binutils-2.34-libbfd-fix.patch

```diff
@@ -0,0 +1,30 @@
+--- lookup-symbol.c.original	2021-08-05 22:21:15.819998378 +0300
```

Comment:
Please add as a patch inline comment a URL to the upstream commit fixing this, so we know for further releases when/if we can drop it easily

Reply:
I have added more information to the patch now.